### PR TITLE
Cleanup the README

### PR DESCRIPTION
**Pull request by whatyouhide**
This will hopefully push people over to hexdocs.pm, where detailed documentation can be found.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,77 +18,26 @@ end
...
-# => [<<196, 5>>, <<3, 18, 122, 27, 115>>]
-# ...
-code = Msgpax.unpack!(iodata, %{binary: true})
-# => #Msgpax.Bin<<<3, 18, 122, 27, 115>>>
-```
-
-#### Deriving of the `Msgpax.Packer` protocol
+* Packing and unpacking Elixir terms via [`Msgpax.pack/1`][docs-msgpax-pack-1] and [`Msgpax.unpack/1`][docs-msgpax-unpack-1] (and their `!` bang variants).
+* Unpacking of partial slices of MessagePack-encoded terms via [`Msgpax.unpack_slice/1`][docs-msgpax-unpack_slice-1].
+* Support for "binary" and "extension" MessagePack types via [`Msgpax.Bin`][docs-msgpax-bin] and [`Msgpax.Ext`][docs-msgpax-ext], respectively.
```
Comment (lexmag) [inline README.md]: In the MessagePack spec they use "Binary" and "Extension" types.

Comment (lexmag) [inline README.md]: Same do we in the module docs.

**Comment by lexmag** [comment]
Maybe we should put `Features` section first?

File: README.md (hunk 1 above)
Comment (whatyouhide) [inline README.md]: Whoops, fixing.

**Comment by whatyouhide** [comment]
Mmm, yeah, let's do that. Updated.

**Comment by lexmag** [comment]
:shipit:

**Pull request merged**